study.md

```diff
@@ -25,7 +25,7 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+A single page application is an application that uses only one markup page and has no page refreshes.  The idea behind this is that the user can seamlessly do everything the app does all on one page with no interruptions, refreshes or reloads.  For this to work, there has to be a large emphasis on client side functionality and a wide use of AJAX to handle any server-related requests.  According to the Single Page App Book, SPAs can redraw any part of the UI without a server roundtrip to get new HTML pages.  They also separate data from presentation by "having a model layer that handles data and a view layer that reads from the models."
```

Comment:
Awesome explanation. Careful with this though `large emphasis on client side functionality and a wide use of AJAX to handle any server-related requests.`

One main point of SPA's is that they tend to make one request (AJAX typically) to the server to get all the data back, then on the front end you can decide what chunks to present to the user without having to make more requests to the server.